### PR TITLE
feature/COR-1684-archive-reproduction-number

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard21**
## Summary

* Moved 'Reproductiegetal' page in the archived pages section, at the top of the archived pages list.
* Changed the section sub-title.
* Added warning tile.
* Any metrics are marked as archived as per the new archive system.

#### Before
<details>
<summary>Toggle before screenshots</summary>

Scherm­afbeelding 2023-07-06 om 14 53 33

</details>

#### After
<details>
<summary>Toggle after screenshots</summary>

Scherm­afbeelding 2023-07-06 om 14 53 10

</details>

### CODE REVIEW

File: packages/app/src/pages/landelijk/reproductiegetal.tsx [hunk 1]
```diff
@@ -125,7 +131,7 @@ const ReproductionIndex = (props: StaticProps<typeof getStaticProps>) => {
             </KpiWithIllustrationTile>
           </TwoKpiSection>
 
-          <ReproductionChartTile data={data.reproduction} timelineEvents={getTimelineEvents(content.elements.timeSeries, 'reproduction')} text={textNl} />
+          <ReproductionChartTile data={reproductionValues} timelineEvents={getTimelineEvents(content.elements.timeSeries, 'reproduction_archived_20230711')} text={textNl} />
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/pages/landelijk/reproductiegetal.tsx:134]: If i'm not wrong, this new timeline has to be created in Sanity. Have you done that as well?

**Review by VWSCoronaDashboard28 — COMMENTED** [review]
Looks good, but please see my comment

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Nice! Good stuff!

**Review by VWSCoronaDashboard27 — DISMISSED** [review]

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Nice that you removed this unused timeline!

**Review by VWSCoronaDashboard28 — APPROVED** [review]
Awesome!

File: packages/app/src/pages/landelijk/reproductiegetal.tsx (hunk 1 above)
Comment (VWSCoronaDashboard28) [reply packages/app/src/pages/landelijk/reproductiegetal.tsx]: I will resolve this as the timeline has been removed.

**Pull request merged**